packages/data-stores/src/products-list/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface ProductsListItem {
+	available: boolean;
+	combined_cost_display: string;
+	cost: number;
+	currency_code: string;
+	description: string;
+	is_domain_registration: boolean;
+	price_tier_list: any;
+	price_tier_lug: string;
+	price_tier_usage_quantity: number | null;
+	price_tiers: any;
+	product_id: number;
+	product_name: string;
+	product_slug: string;
+	product_term: string;
+	product_type: string;
+}
+
+export type ProductsList = Record< string, ProductsListItem >;
+
+export interface ProductsListFailure {
+	message: string;
+}
+
```

Comment:
Not sure, but I think this should be:
```suggestion
interface CustomError extends Error {}
```
 then you won't need the `err as ProductsListFailure` in the resolver.

Reply:
That doesn't seem to work the way you've outlined it. Can you give me any more details?

Reply:
Fair. I haven't tested it! I don't want to waste your time further, let's keep it as you wrote it.